doc/cc65.sgml

```diff
@@ -792,6 +792,21 @@ This cc65 version has some extensions to the ISO C standard.
         size zero, even if it is initialized.
         <p>
 
+<item>  cc65 supports <tt/_Static_assert/ from C11 and C2X.  This is similar
+        to <tt/#error/ but happens at a later stage of translation, so types
+        can be used.
+
+        <tscreen><verb>
+        /* C11 version with message. */
+        _Static_assert (sizeof (int) == 2, "Expected 2-bytes ints.");
+
+        /* C2X version without message. */
+        _Static_assert (sizeof (int) == 2);
+        </verb></tscreen>
+
+        <tt/_Static_assert/ is also available as the macro <tt/static_assert/ in
+        <tt/&lt;assert.h&gt;/.
```

Comment:
It seems to me, it no usual in the cc65 doc to put the `<` `>` around header file names. E.g. https://github.com/cc65/cc65/blob/master/doc/funcref.sgml#L3899

Reply:
Fixed.